File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.MessagingService.Models/message-thread-message_item.ts



export class MessageThreadMessage_Item  {
  
	public MessageThreadMessageID?: number;
	public MessageThreadID: number = 0;
	public SenderAppUserID: number = 0;
	public SenderAppUserFirst: string = "";
	public SenderAppUserMiddle: string = "";
	public SenderAppUserLast: string = "";
	public SentTime: Date = new Date(0);
	public Message: string = "";
	public AttachmentFileID?: number;
	public AttachmentFileName: string = "";
	public AttachmentFileLocation: string = "";
	public AttachmentFileHash: number[] = null;
	public AttachmentFileThumbnailPath: string = "";
	public AttachmentFileSize?: number;
	public AttachmentFileTypeID?: number;
	public AttachmentFileType: string = "";
  
}

File: GTTS/INL.MessagingService/Models/MessageThreadMessage_Item.cs
using System;

namespace INL.MessagingService.Models
{
    public class MessageThreadMessage_Item
    {
        public long? MessageThreadMessageID { get; set; }
        public int MessageThreadID { get; set; }
        public int SenderAppUserID { get; set; }
        public string SenderAppUserFirst { get; set; }
        public string SenderAppUserMiddle { get; set; }
        public string SenderAppUserLast { get; set; }
        public DateTime SentTime { get; set; }
        public string Message { get; set; }
        public long? AttachmentFileID { get; set; }
        public string AttachmentFileName { get; set; }
        public string AttachmentFileLocation { get; set; }
        public byte[] AttachmentFileHash { get; set; }
        public string AttachmentFileThumbnailPath { get; set; }
        public long? AttachmentFileSize { get; set; }
        public long? AttachmentFileTypeID { get; set; }
        public string AttachmentFileType { get; set; }
    }
}
